apps/embed-iframe/package.json

```diff
@@ -34,6 +34,7 @@
     "@testing-library/user-event": "14.5.2",
     "@toruslabs/customauth": "16.0.6",
     "@trilitech-umami/umami-embed": "^0.0.4",
+    "@umami/social-auth": "workspace:*",
```

Comment:
it's simpler to use "workspace:^" because when you run `yarn add @umami/social-auth` it does exactly that, while manually adding dependencies to the package file may be quite tedious
though both ^ and * mean the same in the current context